.rspec

```diff
@@ -1 +1,2 @@
 --require spec_helper
+--format documentation
```

Comment:
Changes RSpec output in terminal. I think it makes it more readable, but ultimately doesn't matter.